routes/eventsManager.js

```diff
@@ -0,0 +1,167 @@
+const EventModel = require('../models/event');
+
+function getAll(){
+    return new Promise( function (resolve, reject) {
```

Comment:
You don't really need to create a Promise here. Mongo does that for you already:

```
Event.find({}, (err, events) => {
    if (err) {
      res.status(404).send(err)
    } else {
      res.status(200).send(events)
    }
  });
```